modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -3,35 +3,51 @@
 [[manage-aliases-standard-databases]]
 = Managing database aliases for standard databases
 
-There are two kinds of database aliases: local and remote.
-A local database alias can only target a database within the same DBMS.
-A remote database alias may target a database from another Neo4j DBMS.
-When a query is run against a database alias, it will be redirected to the target database.
-The home database for users can be set to an alias, which will be resolved to the target database on use.
-Starting with Neo4j 2025.04, a database alias can also be set as the default database.
+Database aliases can be created and managed using a set of Cypher administration commands executed against the `system` database.
+The required privileges are described in the xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[The DBMS ALIAS MANAGEMENT privileges].
+When connected to the DBMS over Bolt, administration commands are automatically routed to the `system` database.
 
-This page describes managing database aliases for standard databases.
-Local and remote database aliases can also be created as part of a xref:database-administration/composite-databases/concepts.adoc[composite database].
-For more information, see xref:database-administration/aliases/manage-aliases-composite-databases.adoc[].
+[NOTE]
+====
+If a transaction modifies a database alias, other transactions concurrently executing against that alias may be aborted and rolled back for safety.
+This prevents issues such as a transaction executing against multiple target databases for the same alias.
+====
+
+There are two kinds of database aliases - local and remote:
 
-A local database alias can be used in all other Cypher commands in place of the target database.
+Local database aliases::
+A local database alias can only target a database within the same DBMS.
+It can be used in all Cypher commands in place of the target database.
 Please note that the local database alias will be resolved while executing the command.
-Privileges are defined on the database, and not the local database alias.
+Privileges are defined on the target database, and not the local database alias.
+
+
+Remote database aliases::
+A remote database alias may target a database from another Neo4j DBMS.
+It can be used for:
+* Connecting to a database of a remote Neo4j DBMS
+* `USE` clauses
+* Setting a user's home database
```

Comment:
should we mention home database for local aliases as well? Feels odd to just have it on the remote side

alternatively we don't mention it on either as we below have `The home database for users can be set to an alias, which will be resolved to the target database on use.` which is true for both local and remote aliases I'd assume.

Copying over https://github.com/neo4j/docs-operations/pull/2372#discussion_r2169118539